tests/test_chart_placeholder.py

```
@@ -1,330 +1,84 @@
+        # Should render chart (not show placeholder text)
+        assert "No data" not in content
+        assert "No jobs yet" not in content
+        # Should have newlines (multi-line chart)
+        assert "\n" in content or len(content) > 0
+
+
+@pytest.mark.asyncio
+async def test_chart_placeholder_has_chart_display():
+    """ChartPlaceholder should have a chart-display element."""
+    app = ChartPlaceholderTestApp()
+    async with app.run_test() as pilot:
```

Ruff (0.14.14)

Local variable `pilot` is assigned to but never used

Remove assignment to unused variable `pilot`

(F841)